modules/search/instagram.py

```diff
@@ -19,76 +19,138 @@
 
 meta = {
 	'name': 'Instagram Search',
-	'author': 'Aman Singh',
+	'author': 'Aman Singh, Rishabh Jain',
 	'version': '0.2',
 	'description': 'Search your query in the Instagram and show the results.',
-	'sources': ('google', 'carrot2', 'bing', 'yippy', 'yahoo', 'millionshort', 'qwant', 'duckduckgo'),
+	'sources': ('google', 'carrot2', 'bing', 'yippy', 'yahoo', 'millionshort', 'qwant', 'duckduckgo', 'instagram'),
 	'options': (
 		('query', None, True, 'Query string', '-q', 'store', str),
 		('limit', 1, False, 'Search limit(number of pages, default=1)', '-l', 'store', int),
 		('count', 50, False, 'Number of links per page(min=10, max=100, default=50)', '-c', 'store', int),
+		('engine', 'google,instagram', False, 'Engine names for search(default=google, instagram)', '-e', 'store', str),
+		('session_id', None, False, 'Insta Account Session_id for more details results (default="")', '-s', 'store', str),
```

Comment:
for more detailed results maybe?

Reply:
basically to extract followers and following 👍

Reply:
@z3y50n you will not get any error of "too many tries" on using session_id